bms_node:
  ros__parameters:
    port_name: "/dev/ttyUSB0"  # 你的串口设备号，根据实际情况修改
    baud_rate: 115200          # 协议文档规定默认波特率 [cite: 28]
    poll_rate: 1.0             # 查询频率 (Hz)
    frame_id: "battery_link"   # TF 坐标系名称
    timeout_ms: 200            # 串口读取超时时间